sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/SourceOptions.scala

```diff
@@ -47,4 +50,6 @@ object SourceOptions {
   val RESPECT_SPARK_SCHEMA = "respectSparkSchema"
   val DEFAULT_RESPECT_SPARK_SCHEMA = false
 
+  val PARALLEL = "parallel"
```

Comment:
I think we should change it to something like `partitionNum`

Reply:
and would have to document at https://github.com/apache/spark/blob/master/docs/sql-data-sources-generic-options.md

Reply:
Yes. Oracle use `PARALLEL`.